sdk/daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/UtilityDamlPrimPackages.scala

```diff
@@ -0,0 +1,1286 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.daml.lf.language
+
+import com.digitalasset.daml.lf.data.Ref._
+
+object UtilityDamlPrimPackages {
+  def check(pkgId: PackageId) = prims.contains(pkgId);
```

Comment:
Could you please add a comment explaining here what this is for and why we had to do this?

Reply:
done

Reply:
Perfect, thanks!